src/pudl_zenodo_storage/__init__.py

```diff
@@ -7,6 +7,7 @@
 import pudl_zenodo_storage.frictionless.eia861
 import pudl_zenodo_storage.frictionless.eia923
 import pudl_zenodo_storage.frictionless.eipinfrastructure
+import pudl_zenodo_storage.frictionless.epa_eia_crosswalk
```

Comment:
The naming on this dataset feels pretty generic. Both EPA and EIA publish dozens (hundreds?) of different datasets. It's not clear from the name which data is being linked.  Isn't it a pretty specific connection, between the EPA CEMS and EIA plants/units/generators/boilers which are defined in a mix of eia860 and eia923?

Reply:
That's a fair point. Could be called `cems_unitid_to_eia_plant_part`. What do you think?

Reply:
or `epacems_unitid_eia_plant_part_crosswalk`

Reply:
Maybe `epacems_unitid_eia_plant_crosswalk`? I don't think we should say `plant_part` or `eia_plant_part` since that already means something specific (and confusing) in the context of the `plant_parts_eia` work.

The `eia_plant` portion wouldn't as specific as it could be, but I think it's specific enough to keep it from getting confused with the many other potential EIA entities.

Whatever more specific name gets used here should be used in the `pudl-scrapers` repo as well.

Reply:
I chose `epacems_unitid_eia_plant_crosswalk`